codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/rustlang/RustReservedWords.kt

```diff
@@ -33,34 +44,19 @@ class RustReservedWordSymbolProvider(private val base: RustSymbolProvider) : Wra
         val reservedWordReplacedName = internal.toMemberName(shape)
         val container = model.expectShape(shape.container)
         return when {
-            container is StructureShape -> when (baseName) {
-                "build" -> "build_value"
-                "builder" -> "builder_value"
-                "default" -> "default_value"
-                "send" -> "send_value"
-                // To avoid conflicts with the `make_operation` and `presigned` functions on generated inputs
-                "make_operation" -> "make_operation_value"
-                "presigned" -> "presigned_value"
-                "customize" -> "customize_value"
-                // To avoid conflicts with the error metadata `meta` field
-                "meta" -> "meta_value"
-                else -> reservedWordReplacedName
+            container is StructureShape -> when (val mapped = reservedWordConfig.structMemberMap[baseName]) {
+                null -> reservedWordReplacedName
+                else -> mapped
             }
```

Comment:
```suggestion
            container is StructureShape ->
                reservedWordConfig.structMemberMap.getOrDefault(baseName, reservedWordReplacedName)
```

These can be slightly simplified.